zuul.d/project-templates.yaml

```diff
@@ -12,6 +12,9 @@
             dependencies:
               - openstack-k8s-operators-content-provider
         - cifmw-crc-podified-edpm-baremetal: *content_provider
+        - periodic-podified-multinode-edpm-deployment-master-tempest: *content_provider
+        - periodic-podified-multinode-edpm-deployment-antelope-tempest: *content_provider
+        - podified-multinode-edpm-deployment-crc-tempest: *content_provider
```

Comment:
thanks - pls add just the check and not the periodic in template

Reply:
done